vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -80,6 +84,10 @@ def parse_pod(pod)
         :restart_policy => pod_restart_policy,
         :dns_policy     => pod.spec.dnsPolicy
       )
+      if pod.spec.respond_to?(:host)
+        new_result[:container_node] = @data_index.fetch_path(
+          :container_nodes, :by_name, pod.spec.host)
+      end
```

Comment:
If the spec doesn't respond to host, and the database already has a container node attached...then should we remove the container node in the database?  If so, this code doesn't handle that condition.